doc/examples/Organization.rst

```diff
@@ -0,0 +1,47 @@
+
+Organization
+==========
+
+Get repository's in an organization
+
+.. code-block:: python
+
+	>>> git = Github(access_token)
+	>>> org = git.get_organization("PyGithub")
+	>>> repository  = org.get_repos()
+	>>> for repos in repository:
+			print(repos)
```

Comment:
It looks like the indentation is off when you preview the change
https://github.com/PyGithub/PyGithub/blob/4ac64d1b21b3aefe20c029003583e1dd0457fe0b/doc/examples/Organization.rst

Reply:
just seen the preview seems different to my copy. I will make the required changes